app/models/patron.rb

```diff
@@ -218,30 +218,17 @@ def can_pay_fines?
     true
   end
 
-  def can_schedule_green_access?
+  def can_schedule_access?(library)
```

Comment:
`can_schedule_access?` and `can_schedule_access?` now have the same user groups.  One of them could theoretically call the other, or a shared method.  Happy to add that refactor, but stopped just short of doing so.